components/scrapeninja/actions/non-js-scraping/non-js-scraping.mjs

```diff
@@ -0,0 +1,106 @@
+import { ConfigurationError } from "@pipedream/platform";
+import { parseObject } from "../../common/utils.mjs";
+import scrapeninja from "../../scrapeninja.app.mjs";
+
+export default {
+  key: "scrapeninja-non-js-scraping",
+  name: "Non-JS Scraping",
```

Comment:
Following the naming guidelines for actions, I think "Scrape without JS" or something similar would be more appropriate (with the verb "Scrape")